app/views/results/common/_annotations.js.erb

```diff
@@ -133,10 +133,10 @@
     var line_end     = line_manager.getLineNumber(focus_node);
 
     // If we selected an entire line the above returns + 1, a fix follows
-    if (mouseSelection.anchorNode.nodeName === "LI"){
+    if (mouseSelection.anchorNode.nodeName === "LI") {
```

Comment:
I'm surprised you didn't change to single-quotes here (since you're improving style in this one too)